components/pedigree/resources/src/main/resources/pedigree/view/exportSelector.js

```diff
@@ -255,11 +255,17 @@ define([
                 var label = new Element('label', {'class': 'export-config-header ped-header'}).insert("Which of the following should be reflected in the affected column in PED file? ");
                 pedContainer.insert(label.wrap('td').wrap('tr', {'class' : "ped-special-options"}));
 
-                hasDisorders && this._addPedOption("disorders", disorders, pedContainer);
-                hasPhenotypes && this._addPedOption("phenotypes", hpos, pedContainer);
-                hasCancers && this._addPedOption("cancers", cancers, pedContainer);
-                hasCandidateGenes && this._addPedOption("candidateGenes", candidateGenes, pedContainer, "candidate genes");
-                hasCausalGenes && this._addPedOption("causalGenes", causalGenes, pedContainer, "confirmed causal genes");
+                var traitsContainner = new Element('table');
+
+                hasDisorders && this._addPedOption("disorders", disorders, traitsContainner);
+                hasPhenotypes && this._addPedOption("phenotypes", hpos, traitsContainner);
+                hasCancers && this._addPedOption("cancers", cancers, traitsContainner);
+                hasCandidateGenes && this._addPedOption("candidateGenes", candidateGenes, traitsContainner, "candidate genes");
+                hasCausalGenes && this._addPedOption("causalGenes", causalGenes, traitsContainner, "confirmed causal genes");
+
+                pedContainer.insert(traitsContainner.wrap('div', {'class': 'scrollable-container', 'style': 'max-height:'+Math.floor(window.innerHeight/2)+'px;'})
```

Comment:
Setting the max height to be half of the viewport isn't a reliable way to solve the problem. There's no guarantee that the rest of the dialog will fit in the other half. If we're going to try to fit the whole dialog on the screen, then a better option is to:
* get the modal's height and the dcrollable container's height when fully displayed
* compute how big the scrollable container needs to be so that the modal can fit on the screen (something like `sHeight - Math.max (0, modalHeight - viewportHeight)`)
* adjust the height of  the scrollable container to this value

Disadvantages:
- you'd need to recompute on each window resize
- on REALLY small screens it will be impossible to fit because the rest of the modal is already too big

There are still options:
- make the whole content of the modal scrollable, not just that part (while keeping the modal fixed & fully on screen)
- make the modal not fixed, allowing it to scroll with the background content.